prover/vk_setup_data_generator_server_fri/src/commitment_utils.rs

```diff
@@ -33,14 +34,26 @@ fn circuit_commitments() -> anyhow::Result<L1VerifierConfig> {
             // The base layer commitment is not used in the FRI prover verification.
             recursion_circuits_set_vks_hash: H256::zero(),
         },
-        recursion_scheduler_level_vk_hash: H256::from_str(&commitments.scheduler)
-            .context("invalid scheduler commitment")?,
+        // Instead of loading the FRI scheduler verification key here,
+        // we load the SNARK-wrapper verification key.
+        // This is due to the fact that these keys are used only for picking the
+        // prover jobs / witgen jobs from the DB. The keys are matched with the ones in
+        // `prover_protocol_versions` table, which has the SNARK-wrapper verification key.
+        // This is OK because if the FRI VK changes, the SNARK-wrapper VK will change as well.
+        // You can actually compute the SNARK-wrapper VK from the FRI VK, but this is not yet
+        // implemented in the `zkevm_test_harness`, so instead we're loading it from a separate
+        // file.
+        recursion_scheduler_level_vk_hash: H256::from_str(&snark_wrapper_vk)
+            .context("invalid SNARK wrapper VK")?,
     })
 }
 
+pub get_snark_vk_hash() -> anyhow::Result<H256> {
+    let snark_vk = get_snark_vk()?;
```

Comment:
does this compile?

Reply:
Already removed, was a merge artifact or something